### PR TITLE
chore: Prepare release

### PR DESCRIPTION
**Pull request by TimePrinciple**
### Summary of the PR

- virtio-bindings: Prepare v0.2.4 release
- virtio-queue: Prepare v0.14.0 release
- virtio-queue-ser: Prepare v0.11.0 release
- virtio-vsock: Prepare v0.8.0 release

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR have Signed-Off-By trailers (with
  `git commit -s`), and the commit message has max 60 characters for the
  summary and max 75 characters for each description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] All added/changed public-facing functionality has entries in the "Upcoming 
  Release" section of CHANGELOG.md (if no such section exists, please create one).
- [x] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

**Comment by TimePrinciple** [comment]
Should we consider dropping the `version` specification after those `path` dependencies :)

File: virtio-queue/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Upcoming
 
+# v0.13.0
```
Comment (stefano-garzarella) [inline virtio-queue/CHANGELOG.md]: little typo here: v0.14.0

**Comment by stefano-garzarella** [comment]
> Should we consider dropping the `version` specification after those `path` dependencies :)

IIRC it was needed for `cargo publish, so I just tried to remove some versions in virtio-vsock:
```diff
diff --git a/virtio-vsock/Cargo.toml b/virtio-vsock/Cargo.toml
index 71d51f1..402938a 100644
--- a/virtio-vsock/Cargo.toml
+++ b/virtio-vsock/Cargo.toml
@@ -11,10 +11,10 @@ edition = "2021"
 
 [dependencies]
 # The `path` part gets stripped when publishing the crate.
-virtio-queue = { path = "../virtio-queue", version = "0.14.0" }
-virtio-bindings = { path = "../virtio-bindings", version = "0.2.4" }
+virtio-queue = { path = "../virtio-queue" }
+virtio-bindings = { path = "../virtio-bindings" }
 vm-memory = { workspace = true }
 
 [dev-dependencies]
-virtio-queue = { path = "../virtio-queue", version = "0.14.0", features = ["test-utils"] }
+virtio-queue = { path = "../virtio-queue", features = ["test-utils"] }
 vm-memory = { workspace = true, features = ["backend-mmap", "backend-atomic"] }
```

But then `cargo publish` is not happy:
```
$ cargo publish --dry-run -p virtio-vsock     
    Updating crates.io index
error: all dependencies must have a version specified when publishing.
dependency `virtio-bindings` does not specify a version
Note: The published dependency will use the version from crates.io,
the `path` specification will be removed from the dependency declaration.
```

So for now I think we can't remove them.

**Comment by TimePrinciple** [comment]
> > Should we consider dropping the `version` specification after those `path` dependencies :)
> 
> IIRC it was needed for `cargo publish`, so I just tried to remove some versions in virtio-vsock:
> 
> ```diff
> diff --git a/virtio-vsock/Cargo.toml b/virtio-vsock/Cargo.toml
> index 71d51f1..402938a 100644
> --- a/virtio-vsock/Cargo.toml
> +++ b/virtio-vsock/Cargo.toml
> @@ -11,10 +11,10 @@ edition = "2021"
>  
>  [dependencies]
>  # The `path` part gets stripped when publishing the crate.
> -virtio-queue = { path = "../virtio-queue", version = "0.14.0" }
> -virtio-bindings = { path = "../virtio-bindings", version = "0.2.4" }
> +virtio-queue = { path = "../virtio-queue" }
> +virtio-bindings = { path = "../virtio-bindings" }
>  vm-memory = { workspace = true }
>  
>  [dev-dependencies]
> -virtio-queue = { path = "../virtio-queue", version = "0.14.0", features = ["test-utils"] }
> +virtio-queue = { path = "../virtio-queue", features = ["test-utils"] }
>  vm-memory = { workspace = true, features = ["backend-mmap", "backend-atomic"] }
> ```
> 
> But then `cargo publish` is not happy:
> 
> ```
> $ cargo publish --dry-run -p virtio-vsock     
>     Updating crates.io index
> error: all dependencies must have a version specified when publishing.
> dependency `virtio-bindings` does not specify a version
> Note: The published dependency will use the version from crates.io,
> the `path` specification will be removed from the dependency declaration.
> ```
> 
> So for now I think we can't remove them.

Thanks for you time and explaination 😊, I haven't tried to release crate, I'm not aware of this constraint :(

**Review by rbradford — APPROVED** [review]

**Comment by stefano-garzarella** [comment]
releases published here and on crates.io, thanks!

**Pull request merged**